### ISSUE TITLE
`<:messages>` are not closable

### ISSUE THREAD

**Issue by AlexanderBl4h** [TheCoder4eu/BootsFaces-OSP#150]
Just try following example: 

	<b:messages id="messages" />
	<h:form>
		<b:messages id="messages" />

		<b:inputText required="true" />
		<b:commandButton value="Show message" update="messages messages-b" />
	</h:form>
	<b:messages id="messages-b" />

Klick `Show messages` and you will see three alerts popping up. Those are at three different locations: 
- before the form
- inside the form
- after the form

Neither of them is close able. 

Any quick fix?

**Comment by stephanrauh** [comment]
Quick fix: add an invisible ```<b:alert /```> somewhere on the page:
``` xml
    <b:alert rendered="false" />
		<b:messages id="messages" />
		<h:form>
		    <b:messages id="messages" />
		
		    <b:inputText required="true" />
		    <b:commandButton value="Show message" update="messages messages-b" />
		</h:form>
		<b:messages id="messages-b" />
    </h:body>
```

**Comment by stephanrauh** [comment]
Solved the problem in version 0.8.0.
